Lib/pathlib.py

```diff
@@ -1361,13 +1361,23 @@ def lstat(self):
             self._raise_closed()
         return self._accessor.lstat(self)
 
-    def link_to(self, target):
+    def link_to(self, link):
```

Comment:
You can't change the argument name without continuing support for the old name as that's an API change.